### PR TITLE
bump some of our core dependencies

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

**Comment by debris** [comment]
@paritytech/ci there is something seriously broken with the CI, cause 5146bd01135f8ef0a6a2ba06fa429430cd3777fd didn't trigger build to start

**Comment by kirushik** [comment]
Should be fixed by #7564, we've got our `grep` arguments wrong.
Please update to the latest master after that one is merged.

File: util/version/build.rs [hunk 1]
```diff
@@ -33,5 +33,5 @@ fn main() {
 		pub fn rustc_version() -> &'static str {{
 			\"{}\"
 		}}
-	", rustc_version::version()).as_bytes()).unwrap();
+	", rustc_version::version().unwrap()).as_bytes()).unwrap();
```
Comment (tomusdrw) [inline util/version/build.rs]: Why is it safe?

Comment (debris) [reply util/version/build.rs]: it's not. neither of the `unwraps` here is safe. I will fix it

**Pull request merged**